terminal/terminal.go

```diff
@@ -158,10 +154,10 @@ func (t *Term) promptForInput() (string, error) {
 func (t *Term) handleExit() (int, error) {
 	fullHistoryFile, err := config.GetConfigFilePath(historyFile)
 	if err != nil {
-		fmt.Println("Error saving history file:", err)
+		fmt.Println("Error saving history file: ", err)
```

Comment:
True, so 5 lines belowe is fmt.Println("readline history error: ", err), which has space, I will fix this